assets/charts/components/rook/templates/NOTES.txt

```diff
@@ -3,18 +3,9 @@ The Rook Operator has been installed. Check its status by running:
 
 Visit https://rook.io/docs/rook/master for instructions on how to create and configure Rook clusters
 
-Note: You cannot just create a CephCluster resource, you need to also create a namespace and
-install suitable RBAC roles and role bindings for the cluster. The Rook Operator will not do
-this for you. Sample CephCluster manifest templates that include RBAC resources are available:
-
-- https://rook.github.io/docs/rook/master/ceph-quickstart.html
-- https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/cluster.yaml
-
 Important Notes:
-- The links above are for the unreleased master version, if you deploy a different release you must find matching manifests.
-- You must customise the 'CephCluster' resource at the bottom of the sample manifests to met your situation.
-- Each CephCluster must be deployed to its own namespace, the samples use `rook-ceph` for the cluster.
+- You must customize the 'CephCluster' resource in the sample manifests for your cluster.
+- Each CephCluster must be deployed to its own namespace, the samples use `rook-ceph` for the namespace.
```

Comment:
Q: Is it necessary that these notes should also be part of the component configuration documentation ? Since we take the responsibility of installing charts, the user never actually reads chart notes.

Reply:
Nope, the user's interface is only lokocfg config so we should do the lifting on user's behalf.